src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,85 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+type PaginationType = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (arg0: number) => void,
+};
+
+export const Pagination: React.FC<PaginationType> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const NUMBER_OF_PAGES = Math.ceil(total / perPage);
+  const IF_CURRENT_PAGE_IS_FIRST = currentPage === 1;
+  const IF_CURRENT_PAGE_IS_LAST = currentPage === NUMBER_OF_PAGES;
+
+  const ifPageChange = (page: number) => {
+    if (
+      page !== currentPage
+      && page > 0
+      && page <= NUMBER_OF_PAGES
+    ) {
+      return onPageChange(page);
+    }
+
+    return undefined;
```

Comment:
```suggestion
  const ifPageChange = (page: number) => {
    if (
      page !== currentPage
      && page > 0
      && page <= NUMBER_OF_PAGES
    ) {
      onPageChange(page);
    }
```
onPageChange does not return anything